pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/metrics/MetricConfig.java

```diff
@@ -121,7 +121,7 @@ public static MetricConfig getMetricConfigFrom(DeviceDriverConfig ddrConfig) {
         metricConfig.setStreamWriterIntervalSeconds(Integer.parseInt(ddrConfig.getProperties().getOrDefault(METRIC_STREAM_WRITER_INTERVAL_SECONDS.getName(), METRIC_STREAM_WRITER_INTERVAL_SECONDS.getDefaultValue())));
         metricConfig.setMetricStream(ddrConfig.getProperties().getOrDefault(METRIC_STREAM_NAME.getName() + pscId, METRIC_STREAM_NAME.getDefaultValue() + pscId));
         metricConfig.setControllerURI(URI.create(ddrConfig.getProperties().getOrDefault(METRIC_CONTROLLER_URI.getName(), METRIC_CONTROLLER_URI.getDefaultValue())));
-        metricConfig.setMetricsScope(METRIC_SCOPE_NAME.getDefaultValue());
```

Comment:
please remove the instance variable defined at class level as we dont need it anymore.

Reply:
Done.